src/styles/_components.scss

```diff
@@ -70,6 +71,25 @@
   font-size: $font-size-5;
 }
 
+.maputnik-wide-button {
+  padding: $margin-2 $margin-3;
+}
+
+.maputnik-green-button {
+  background-color: $color-green;
+  color: $color-black;
+}
+
+.maputnik-green-hover-button:hover, .maputnik-green-hover-button:focus {
+  background-color: $color-green;
+  color: $color-white;
```

Comment:
Corrected. :]

Reply:
The button on hover still falls below the required contrast ratio (see http://contrast-ratio.com/#white-on-%2353b971). Screenshot below

<img width="195" alt="screen shot 2018-07-08 at 15 02 14" src="https://user-images.githubusercontent.com/235915/42420594-f375e48a-82bf-11e8-8229-fd49800aeffb.png">

Related to ticket #318. If you have time a fix would be appreciated, otherwise we can merge and fix it up in issue #318

Reply:
Check the following commits, please. :)

Reply:
You're right. Give me a minute. : )

Reply:
Fixed & ready to be merged.